.circleci/config.yml

```diff
@@ -180,7 +153,26 @@ jobs:
           no_output_timeout: "30m"
       - store_artifacts:
           path: /go/src/github.com/Azure/acs-engine/_logs
-  ginkgo-k8s-windows-e2e:
+  k8s-1.7-release-e2e:
+    working_directory: /go/src/github.com/Azure/acs-engine
+    docker:
+      - image: quay.io/deis/go-dev:v1.2.0
+        environment:
+          GOPATH: /go
+    steps:
+      - checkout
+      - run: |
+          echo 'export CLUSTER_DEFINITION=examples/e2e-tests/kubernetes/release/1.7/definition.json' >> $BASH_ENV
```

Comment:
So you dont have to copy these files. Instead you can set `ORCHESTRATOR_RELEASE` to whatever version you want (1.7, 1.8, etc...) and it splice that into the definition before it runs generate. This means we just change the circle configuration and we can test any version we like.